bin/lib/start.js

```diff
@@ -38,6 +38,25 @@ module.exports = function (program, server) {
         })
       }
 
+      // Read in ENV variables
+      options
+        .map(option => {
+          const configName = option.name.replace(/-([a-z])/g, (_, letter) => letter.toUpperCase())
+          const snakeCaseName = configName.replace(/([A-Z])/g, '_$1')
+          const envName = `SOLID_${snakeCaseName.toUpperCase()}`
+
+          return { configName, envName }
+        })
```

Comment:
Above can simply be

```JavaScript
      options.map(({ name }) => ({
        configName: name.replace(/-([a-z])/g, (_, l) => l.toUpperCase()),
        envName: `SOLID_${name.replace(/-/g, '_')}`
      }))
```

Reply:
I believe your example is missing a `. toUpperCase()` on the `envName` line:
```
envName: `SOLID_${name.replace(/-/g, '_').toUpperCase()}`
```

This is a possible solution, though it would map `useCorsProxy` to `SOLID_USECORSPROXY`, where the solution that I submitted would map it to `SOLID_USE_CORS_PROXY` which I thought was better.

Let me know what you prefer.

Reply:
> I believe your example is missing a `. toUpperCase()`

It was; fixed.

> though it would map useCorsProxy to SOLID_USECORSPROXY, 

It would not; note the `g` after `/-/g`.

Reply:
I think there may be an oversight here. [`useCorsProxy`](https://github.com/solid/node-solid-server/blob/master/bin/lib/options.js#L149) does not contain any `-`s in it.

The examples I gave are the output from running the two proposed options.

Reply:
You're right; I wonder whether that's a mistake in that file then, having both properties with dashes and in camel case. But I guess that in your code, you'd want to change the dashes into underscores as well?

Reply:
The way this method is currently written supports the `name`s in `./options` to be in CamelCase or kebab-case, and so I think leaving that as is, might be best for the time being.

![image](https://user-images.githubusercontent.com/207950/47273051-da30f700-d542-11e8-94e6-337f8c4e999b.png)

I agree that having both camel and kebab case `name`s in `./options` is inconsistent, but I would suggest deferring that change for another pull request.

Changing those values will be a breaking change. For example if we were to change `name: 'serverUri',` to `name: 'server-uri',`, anyone currently using the `--serverUri` flag would need to change it to `--server-uri`.

Reply:
I think we should change them and apply the `configName: name.replace(/-([a-z])/g, (_, l) => l.toUpperCase()),` transformation from above. Or maybe the other way round, having them in camel case in the config (since it is JS) and then changing them to hyphenated.

Reply:
Ah wait, never mind… it should've been --server-uri, not --serverUri. Sigh.